Makefile

```diff
@@ -84,11 +84,7 @@ verify-gomod:
 check-style: golangci-lint
 
 golangci-lint:
-# https://stackoverflow.com/a/677212/1027058 (check if a command exists or not)
-	@if ! [ -x "$$(command -v golangci-lint)" ]; then \
-		echo "golangci-lint is not installed. Please see https://github.com/golangci/golangci-lint#install for installation instructions."; \
-		exit 1; \
-	fi; \
+	$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.2
```

Comment:
Should we run this only if golangci-lint is not present?

Reply:
We are moving away from downloading a binary, to installing via the Go toolchain. If you run this command for the 2nd time, it doesn't do anything.

Reply:
Ah ok ok, that makes sense. Thanks!